encoder/vaapiencoder_h264.cpp

```diff
@@ -1313,6 +1334,11 @@ void VaapiEncoderH264::setIdrFrame (const PicturePtr& pic)
     m_idrNum++;
 }
 
+void VaapiEncoderH264::setAUD (const PicturePtr& pic, bool enable)
```

Comment:
@lizhong1008 , I think I have to follow the first solution because I can't get the pointer of VaapiEncoderH264 in VaapiEncPictureH264::getOutput().